### ISSUE TITLE
Instantiate a shell

### ISSUE THREAD

**Issue by samholmes** [gokcehan/lf#896]
`$` will allow you to run a shell command and the output goes somewhere (idk). Is there a way to make `$` open a full shell sub-process (prompt) in the preview window at the current directory? This way all of the output is visible and you get a full log of all the commands your previously ran. Of course, you could run `exit` to close the shell, but it'd be additionally convenient if you could "hide" the shell somehow so you can get back to it later with `$`. 

I don't see why you would ever have to leave `lf` with the ability to: navigate around the file system, pop into the active shell, run some commands, pop out of the shell, move to another directory, pop back into the shell (in the active directory), run some new commands, etc.

**Comment by lahwaacz** [comment]
The default binding for launching a shell is `w`, see the [documentation](https://pkg.go.dev/github.com/gokcehan/lf#section-documentation).

The output of the shell should redraw whenever you exit the shell and start a new one from lf (perhaps this depends on your terminal capabilities), so there should be no need to "hide" an active shell.

**Comment by samholmes** [comment]
Is there a way to have the shell open in the third (right-most) window; where the file previews are shown?

**Comment by lahwaacz** [comment]
No, you can't open the shell "within" the lf window/screen – that's a [non-feature](https://github.com/gokcehan/lf#non-features). You can use [tmux](https://github.com/tmux/tmux) to split the terminal whatever you like.

**Comment by samholmes** [comment]
The issue with using something like tmux is that it doesn't manage the lf screens. I'm interested in managing the behavior of the three windows/screens.

**Comment by lahwaacz** [comment]
If you have a lf binding for splitting the current tmux window, you can combine it with toggling the `preview` option.

**Comment by samholmes** [comment]
Interesting. Can you toggle lf to only show one narrow screen of the current directory? This way I can bind many lf commands to open a new tmux window on the right with a 3/4th screen width (or similar) for not only an interactive shell, but also for showing editors which may be cool too. How do I change the view mode for lf (show only current directory) and then how do I send the correct tmux commands to split the window?

**Comment by lahwaacz** [comment]
You can try something like `:set ratios 1:1:1000` or `1:1000:1`, but unfortunately that depends on whether you have `preview` on or off. I don't think lf has an option to set the number of panes to 2 or 1. As for tmux, you can find some examples [here](https://wiki.archlinux.org/title/Tmux#Alternate_approach_for_default_session) and in other sections on that page.

**Comment by Tony-Sol** [comment]
Sorry for my bumping, but is there way to inherit shell options for `$` - it doesn't follow neither aliases nor env variables, sadly

**Comment by CatsDeservePets** [comment]
> You can try something like `:set ratios 1:1:1000` or `1:1000:1`, but unfortunately that depends on whether you have `preview` on or off. I don't think lf has an option to set the number of panes to 2 or 1. As for tmux, you can find some examples [here](https://wiki.archlinux.org/title/Tmux#Alternate_approach_for_default_session) and in other sections on that page.

That is not true. You use `set ratios 1:1` or even `set nopreview; set ratios 1`.

**Comment by CatsDeservePets** [comment]
> `$` will allow you to run a shell command and the output goes somewhere (idk). Is there a way to make `$` open a full shell sub-process (prompt) in the preview window at the current directory? This way all of the output is visible and you get a full log of all the commands your previously ran. Of course, you could run `exit` to close the shell, but it'd be additionally convenient if you could "hide" the shell somehow so you can get back to it later with `$`.

You can just use `!` instead of `$` and you will see all the output.

<img width="1033" height="801" alt="Image" src="https://github.com/user-attachments/assets/60016a72-5d00-45b2-b9d0-5408e4fc7891" />

**Comment by CatsDeservePets** [comment]
> Sorry for my bumping, but is there way to inherit shell options for `$` - it doesn't follow neither aliases nor env variables, sadly

Hello, there is currently no way to do that. There is still an open issue regarding this https://github.com/gokcehan/lf/issues/467.
